Themes/default/Search.template.php

```diff
@@ -467,37 +414,19 @@ function template_results()
 	</div><!-- $topic[css_class] -->';
 			}
 		}
-
-		echo '
-	<div class="pagesection">
-		<span>', $context['page_index'], '</span>
-	</div>';
 	}
 
-	// Show a jump to box for easy navigation.
 	echo '
-	<br class="clear">
-	<div class="smalltext righttext" id="search_jump_to"></div>
-	<script>';
+	<div class="pagesection">';
 
-	if (!empty($options['display_quick_mod']) && $options['display_quick_mod'] == 1 && !empty($context['topics']) && $context['can_move'])
+	if (!empty($context['topics']))
 		echo '
-		if (typeof(window.XMLHttpRequest) != "undefined")
-			aJumpTo[aJumpTo.length] = new JumpTo({
-				sContainerId: "quick_mod_jump_to",
-				sClassName: "qaction",
-				sJumpToTemplate: "%dropdown_list%",
-				sCurBoardName: "', $context['jump_to']['board_name'], '",
-				sBoardChildLevelIndicator: "==",
-				sBoardPrefix: "=> ",
-				sCatSeparator: "-----------------------------",
-				sCatPrefix: "",
-				bNoRedirect: true,
-				bDisabled: true,
-				sCustomName: "move_to"
-			});';
+		<div class="pagelinks">', $context['page_index'], '</div>';
 
+	// Show a jump to box for easy navigation.
 	echo '
+		<div class="smalltext pagelinks floatright" id="search_jump_to"></div>
+		<script>
```

Comment:
Indenting.

Reply:
Er, sorry, but what's off with this one?

Reply:
I will retract that, i missed the closing.

Reply:
No worries.